content/collections/guides_and_surveys/en/resource-center-overview.md

```diff
@@ -0,0 +1,30 @@
+---
+id: 20749d99-c2d4-4576-80a3-24233897706c
+blueprint: guides_and_survey
+title: Overview
+landing: false
+updated_by: 15756874-6eaf-4a8c-8779-bd4081ba41b6
+updated_at: 1750961121
+this_article_will_help_you:
+  - 'Understand the basic concepts of the Resource Center'
+---
+Resource center is a widget that integrates directly into the [Guides and Surveys SDK](/docs/guides-and-surveys/sdk#content-security-policy-csp). It surfaces insightful and useful information to your users from anywhere on your site. The Resource Center widget appears as a clickable help icon. The center itself provides:
+- Search functionality
+- Recommended content
+- Quick links
+- Call to action button
+
+Content for your Resource Center can be external or publicly available documentation repositories, articles, videos, or third-party chat modules.
+
+{{partial:admonition type="note" heading=""}}
+Resource Center is implemented through the Guides and Surveys SDK. 
```

Comment:
@JeffScattini-amplitude Should call out the Web SDK specifically, and provide a link to it.